manifests/s/sebescudie/GammaLauncher/5.2.0-preview02/sebescudie.GammaLauncher.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: sebescudie.GammaLauncher
+PackageVersion: 5.2.0-preview02
+InstallerType: inno

```

Comment:
```suggestion
InstallerType: inno
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```

Reply:
Thanks for the pointer, will commit that one once the `cimgui` issue is solved.